charts/tidb-cluster/templates/tidb-cluster.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     pingcap.com/pd.{{ template "cluster.name" . }}-pd.sha: {{ include "pd-configmap.data-digest" . | quote }}
     pingcap.com/tikv.{{ template "cluster.name" . }}-tikv.sha: {{ include "tikv-configmap.data-digest" . | quote }}
     pingcap.com/tidb.{{ template "cluster.name" . }}-tidb.sha: {{ include "tidb-configmap.data-digest" . | quote }}
+    topologyKey: {{ .Values.topologyKey }}
```

Comment:
What about change `topologyKey` to `pingcap.com/ha-topology-key`?

and define it as a const like https://github.com/pingcap/tidb-operator/blob/master/pkg/label/label.go#L43 does.

Reply:
updated